src/locale/locales/de.json

```diff
@@ -56,7 +56,7 @@
   },
   "SNTable.SelectionLabel.DeselectedValue": {
     "value": "Die Werteauswahl ist aufgehoben.",
-    "comment": "announced by screen readers  when a value is deselected. (tew 210929)",
+    "comment": "announced by screen readers when a value is deselected. (tew 210929)",
```

Comment:
please don't do irrelevant changes like this to big PRs

Reply:
it is fine for now, but for next time :)

Reply:
Sorry, removed it, and will be up in another PR.